### ISSUE TITLE
extra styling

### ISSUE THREAD

**Issue by franfranfranfran** [leopolicastro/prj-rev-bwfs-tea-cozy#5]
https://github.com/leopolicastro/prj-rev-bwfs-tea-cozy/blob/master/TeaCozy/main.css#L149-L153

try getting rid of all three lines of above-linked footer styling. does anything change about how that section looks? try to avoid leaving in or adding any unnecessary bits of styling.

**Comment by leopolicastro** [comment]
You're right it didn't change anything. Sometimes to get something to display correctly it's a process of trial and error for me and it leads to lines of code that I am not sure if they are or aren't doing anything.
